packages/snap-preact-components/src/components/Atoms/Element/Element.tsx

```diff
@@ -0,0 +1,61 @@
+/** @jsx jsx */
+import { Fragment, h } from 'preact';
+import { jsx, css } from '@emotion/react';
+import classnames from 'classnames';
+import { observer } from 'mobx-react-lite';
+import { Theme, useTheme, CacheProvider } from '../../../providers';
+import { ComponentProps, StylingCSS } from '../../../types';
+import { mergeProps } from '../../../utilities';
+
+const CSS = {
+	Element: () => css({}),
+};
+
+export const Element = observer((properties: ElementProps): JSX.Element => {
+	const globalTheme: Theme = useTheme();
+	const defaultProps: Partial<ElementProps> = {};
+
+	const props = mergeProps('element', globalTheme, defaultProps, properties);
+
+	const { type, content, attributes, disableStyles, className, style } = props;
+
+	const styling: { css?: StylingCSS } = {};
+
+	if (!disableStyles) {
+		styling.css = [CSS.Element(), style];
+	} else if (style) {
+		styling.css = [style];
+	}
+
+	const args = {
+		...styling,
+		...attributes,
+		className: classnames('ss__element', className),
+		children: content,
+		onClick: undefined,
+	};
+
+	return type ? (
+		<CacheProvider>
+			{type == 'div' && <div {...args}></div>}
+
+			{type == 'span' && <span {...args}></span>}
+
+			{type == 'p' && <p {...args}></p>}
+
+			{type == 'label' && <label {...args}></label>}
+		</CacheProvider>
+	) : (
+		<Fragment />
+	);
+});
+
+export interface ElementProps extends ComponentProps {
+	type: 'div' | 'span' | 'p' | 'label';
+	content?: string;
+	attributes?: {
+		[name: string]: string | number | undefined;
+		//no onclicks allowed
+		onClick?: undefined;
```

Comment:
I'm wondering if this preventing onClick is necessary here...
And if so, do we then need to prevent all types of other events as well?

Reply:
This came from our discussion previously about all the bad things someone can do with onclick functions. removing the ability removed custom code. Things with onClicks are not what this component is intended for. And by allowing it will cause people to abuse it and run wild creating their own "components". At least thats what we had decided previously. 

As for the other events, I had a similar thought 🤷  I could see them being dangerous as well.

Reply:
Yea, I remember that discussion, but I'm just not sure there is much point in preventing it here when there are other ways of doing it elsewhere. Also, how many other events would be have to try and prevent here?

Reply:
Let's revisit the need for `onClick` and potentially preventing other events after we experiment more.